modules/command-not-found/init.zsh

```diff
@@ -11,6 +12,9 @@ if [[ -s '/etc/zsh_command_not_found' ]]; then
 # Load command-not-found on Arch Linux-based distributions.
 elif [[ -s '/usr/share/doc/pkgfile/command-not-found.zsh' ]]; then
   source '/usr/share/doc/pkgfile/command-not-found.zsh'
+# Load command-not-found on Mac OS X when homebrew tap is configured.
+elif (( $+commands[brew] )) && [[ "$(brew command command-not-found-init 2> /dev/null)" ]]; then
```

Comment:
It might ever so slightly deviate from some places in prezto. But sure, let's follow what the upstream recommends.